src/diagnostics/define_diagnostics.jl

```diff
@@ -179,130 +181,276 @@ function define_diagnostics!(land_model)
-            compute_soilco2_diffusivity!(out, Y, p, t, land_model),
+            compute_stomatal_conductance!(out, Y, p, t, land_model),
+    )
+	
+        # Canopy transpiration
+    add_diagnostic_variable!(
+        short_name = "trans",
+        long_name = "Canopy Transpiration",
+        standard_name = "canopy_transpiration",
+        units = "m s^-1",
+        comments = "The water evaporated from the canopy due to leaf transpiration.",
+        compute! = (out, Y, p, t) ->
+            compute_canopy_transpiration!(out, Y, p, t, land_model),
+    )	
+	
+	### Canopy - Energy
+	    # Canopy aerodynamic resistance
+    add_diagnostic_variable!(
+        short_name = "crae",
+        long_name = "Canopy Aerodynamic Resistance",
+        standard_name = "canopy_aerodynamic_resistance",
+        units = "s m^-1", # ??? not sure
+        comments = "The canopy aerodynamic resistance. Aerodynamic resistance (r_a) is a measure of how much the air near the Earth's surface resists the movement of water vapor and heat from the surface into the air.", #?? of steam, leaves, roots?
+        compute! = (out, Y, p, t) ->
+            compute_canopy_aerodynamic_resistance!(out, Y, p, t, land_model),
+    )
+	
+        # Canopy latent heat flux
+    add_diagnostic_variable!(
+        short_name = "clhf",
+        long_name = "Canopy Latent Heat Flux",
+        standard_name = "canopy_latent_heat_flux",
+        units = "W m^-2",
+        comments = "The energy used for canopy transpiration.", #?? of steam, leaves, roots?
+        compute! = (out, Y, p, t) ->
+            compute_canopy_latent_heat_flux!(out, Y, p, t, land_model),
     )
 
+        # Canopy sensible heat flux
     add_diagnostic_variable!(
-        short_name = "scms",
-        long_name = "Soil CO2 Microbial Source",
-        standard_name = "soil_co2_microbial_source",
-        units = "", # check
-        comments = "The production of CO2 by microbes in the soil. Vary by layers of soil depth.",
+        short_name = "cshf",
+        long_name = "Canopy Sensible Heat Flux",
+        standard_name = "canopy_sensible_heat_flux",
+        units = "W m^-2",
+        comments = "The energy used for canopy temperature change.", #?? of steam, leaves, roots?
         compute! = (out, Y, p, t) ->
-            compute_soilco2_source_microbe!(out, Y, p, t, land_model),
+            compute_canopy_sensible_heat_flux!(out, Y, p, t, land_model),
     )
 
+    ### Canopy - Hydraulics
+        # Canopy water potential
     add_diagnostic_variable!(
-        short_name = "gs",
-        long_name = "Stomatal Conductance",
-        standard_name = "stomatal_conductance",
-        units = "m s^-1",
-        comments = "The conductance of leaves. This depends on stomatal opening. It varies with factors such as soil moisture or atmospheric water demand.",
+        short_name = "cwp",
+        long_name = "Canopy Water Potential",
+        standard_name = "canopy_water_potential",
+        units = "Pa",
```

Comment:
this one is measured in `m` actually - same as soil -  but we get Pa when needed by multiplying by rho_l *g